civicrm/custom/php/CRM/NYSS/BAO/Integration/WebsiteEvent/SurveyEvent.php

```diff
@@ -44,19 +44,46 @@ public function process(int $contact_id): static {
     return $this;
   }
 
+  /**
+   * @param int $contact_id
+   *
+   * @return void
+   * @throws \CRM_Core_Exception
+   * @throws \Civi\API\Exception\UnauthorizedException
+   */
   protected function processForm(int $contact_id): void {
-    // form_id, form_values, form_title, detail??
-    $params = new stdClass();
-    $params->form_id = $this->getFormId();
-    $params->form_values = $this->getFormFields();
-    $params->form_title = $this->getFormTitle();
-    $params->detail = $this->getEventDetails();
-    $params->created_at = $this->getEventData()->getCreatedAtAsDateTime();
-
-    $result = CRM_NYSS_BAO_Integration_Website::processSurvey($contact_id, $this->getEventAction(), $params);
-    if ($result['is_error'] == 1) {
-      throw new Exception("Error processing webform/survey: " . $result['error_message']);
-    }
+
+    // NYSS 16799 -- No more custom field groups for surveys. Now, we just add
```

Comment:
rather than put all the business logic here, why don't we just update CRM_NYSS_BAO_Integration_Website::processSurvey() to reflect the new method of storing data?